mds/enums.py

```diff
@@ -26,6 +26,9 @@ def choices(enum):
 DEVICE_CATEGORY = enum.Enum(
     "Device category",
     [
+        ("bicycle", pgettext_lazy("Device category", "Bicycle")),
+        # temporary bike line in order to make the export button work while the elastic DB is still not updated for bicycle.
```

Comment:
Should we keep this line ? This temporary fix is not needed if we can sync the changes of databases to ES quickly. What do you think @bors-ltd  ?